[1.1.1.1] Update DNS over Discord docs to include changing DNS provider (#4854)
cloudflare/cloudflare-docs: content/1.1.1.1/other-ways-to-use-1.1.1.1/dns-over-discord.md

The refresh button is available on all responses to the `/dig` command, including those that resulted in an error, such as an unknown domain or no records found.

### Changing DNS provider

> By default, the DNS over Discord bot will use Cloudflare's 1.1.1.1 DNS service. You can run the DNS lookup with alternate DNS providers by using the dropdown below the result. Clicking it will show a list of available providers, and selecting a new provider will update the results in the message. Any user can change the DNS provider.
> 
> Like the refresh button, the provider dropdown is available on all responses to the `/dig` command.
> 
> <details>
> <summary>Supported DNS providers</summary>
> <div>
> 
> * [1.1.1.1 (Cloudflare)](https://developers.cloudflare.com/1.1.1.1/)
> * [1.1.1.2 (Cloudflare Malware Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
> * [1.1.1.3 (Cloudflare Malware + Adult Content Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
> * [8.8.8.8 (Google)](https://developers.google.com/speed/public-dns)
> * [9.9.9.9 (Quad9)](https://www.quad9.net/)

Reviewer: ```suggestion
By default, the DNS over Discord bot uses Cloudflare's 1.1.1.1 DNS service. You can run the DNS lookup with alternate DNS providers by selecting the dropdown below the result. This shows you a list of available providers. Selecting a new provider updates the results in the message. Any user can change the DNS provider.
```

As with the `/dig` command, any user can press the refresh button to refresh the displayed DNS results, including for DNS queries that had previously failed.

### Changing DNS provider

> Much like the `/dig` command, the `/multi-dig` command also allows for the DNS provider being used for the lookup to be changed. The menu will appear after each set of DNS results (or after each block of results if more than 10 record types are requested).
> 
> This menu can be used be any user to change the DNS provider used for the lookup, like with the `/dig` command.

Reviewer: ```suggestion
Like the `/dig` command, you can change the DNS provider when using the `/multi-dig` command. The menu appears after each set of DNS results (or after each block of results if more than 10 record types are requested).

This menu can be used be any user to change the DNS provider used for the lookup.
```

> <details>
> <summary>Supported DNS providers</summary>
> <div>
> 
> * [1.1.1.1 (Cloudflare)](https://developers.cloudflare.com/1.1.1.1/)
> * [1.1.1.2 (Cloudflare Malware Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
> * [1.1.1.3 (Cloudflare Malware + Adult Content Blocking)](https://developers.cloudflare.com/1.1.1.1/setup/#1111-for-families)
> * [8.8.8.8 (Google)](https://developers.google.com/speed/public-dns)
> * [9.9.9.9 (Quad9)](https://www.quad9.net/)
> 
> </div>
> </details>

Reviewer: ```suggestion
```

### `privacy` command

> The `/privacy` command displays the Privacy Policy notice for using the 1.1.1.1 DNS over Discord bot. This notice can also be viewed at [https://dns-over-discord.v4.wtf/privacy](https://dns-over-discord.v4.wtf/privacy).

Reviewer: ```suggestion
The `/privacy` command displays the Privacy Policy notice for using the 1.1.1.1 DNS over Discord bot. You can also [refer to the Privacy Policy page](https://dns-over-discord.v4.wtf/privacy) to access it.
```

Example:

```txt
/privacy
```

### `terms` command

> The `/terms` command displays the Terms of Service notice for using the 1.1.1.1 DNS over Discord bot. This notice can also be viewed at [https://dns-over-discord.v4.wtf/terms](https://dns-over-discord.v4.wtf/terms).

Reviewer: ```suggestion
The `/terms` command displays the Terms of Service notice for using the 1.1.1.1 DNS over Discord bot. You can also [refer to the Terms of Service page](https://dns-over-discord.v4.wtf/terms) to access it.
```
